acme/dns_challenge.go

```diff
@@ -124,19 +202,20 @@ func unFqdn(name string) string {
 	return name
 }
 
-// waitFor polls the given function 'f', once per second, up to 'timeout' seconds.
-func waitFor(timeout int, f func() (bool, error)) error {
+// waitFor polls the given function 'f', once every 'interval' seconds, up to 'timeout' seconds.
+func waitFor(timeout, interval int, f func() (bool, error)) error {
```

Comment:
Yes, limitReader for example. I'm also thinking about crypto.go as there are multiple functions in there which are mere utility functions.